lib/queue.js

```diff
@@ -223,6 +223,91 @@ Queue.prototype.checkHealth = function (cb) {
   return promise;
 };
 
+Queue.prototype._scanForJobs = function(cursor, set, cb) {
+  // Ideally, we would be able to paginate redis sets like we do sorted ranges.
+  // For an MVP, however, we're aiming to replace our usage of Arena for Bull 1.x
+  // and Bull 1.x calls SMEMBERS -- which is more blocking than SSCAN.
+
+  this.client.sscan(
+    this.toKey('failed'),
+    cursor,
+    'COUNT', '100',
```

Comment:
I picked an arbitrary value here, but it doesn't matter here because this scans through the entire set. When we [add SET pagination](https://github.com/mixmaxhq/bee-queue/issues/3) this will be solved for free.